doc/source/index.rst

```diff
@@ -2,7 +2,7 @@ Welcome to SNMF's Documentation!
 ====================================
 
 ``SNMF``: This library implements the stretched non negative matrix factorization (sNMF) and sparse stretched NMF
-(ssNMF) algorithms described in ...
+(ssNMF) algorithms.
 
 This algorithm is designed to do an NMF factorization on a set of signals ignoring any uniform stretching of the signal
```

Comment:
Actually this looks good here, can't we just use this in the readme?

Reply:
Yes, pasted.